contracts/test/v0.8/foundry/vrf/VRFV2Plus.t.sol

```diff
@@ -924,5 +930,169 @@ contract VRFV2Plus is BaseTest {
     );
     assertNotEq(requestId, requestId2);
     assertNotEq(preSeed, preSeed2);
+    assertTrue(s_testCoordinator.pendingRequestExists(subId));
+  }
+
+  function testRequestRandomWords_MultipleConsumers_PendingRequestExists() public {
```

Comment:
Do we have a test to check what happens when you have pending requests and you try to remove the consumer or cancel an entire subscription? Do we need it?

Reply:
Doesn't look like we have those tests. I can add those as follow up tests